scio-examples/src/main/scala/com/spotify/scio/examples/extra/JavaConvertersExample.scala

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright 2018 Spotify AB.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package com.spotify.scio.examples.extra
+
+import org.apache.beam.sdk.io.{AvroIO, TextIO}
+import org.apache.beam.sdk.options.ValueProvider.StaticValueProvider
+import com.spotify.scio.JavaConverters._
+
+object JavaConvertersExample {
```

Comment:
Would be nice to have test together with (even experimental) PR, preferably as part of some dummy pipeline.

Reply:
Absolutely, will do that next.

Reply:
Done.

Not that the pipeline test doesn't actually test whether the correct file was written to. In test mode, `.saveAsCustomOutput` never actually applies to the transform. It just compares the SCollection immediately prior to output to the expected collection.

So the dummy pipeline mostly provides an example for users trying to understand the Java Converters. From a testing standpoint, the pipeline test just checks that the code compiles. If we want to makes assertions on the values returned by implicit conversions, we'd need a regular unit test.